### PR TITLE
Immediate hydration on page load

### PR DESCRIPTION
**Pull request by csjh**
### Description

Adds a `backgroundFetch` function and `longQueryNotifier` to querystore

toast notification shows for 3 seconds after query takes longer than 1 second to execute:
image

### Checklist

- [x] For UI or styling changes, I have added a screenshot or gif showing before & after
- [x] I have added a [changeset](https://github.com/evidence-dev/evidence/blob/main/CONTRIBUTING.md#adding-a-changeset)

### CODE REVIEW

File: packages/preprocess/src/process-queries.cjs [hunk 1]
```diff
@@ -261,6 +267,15 @@ const createDefaultProps = function (filename, componentDevelopmentMode, duckdbQ
 			}, 5000);
 		};
 
+		const longQueryNotifier = (info) => {
+			toasts.add({
+				id: Math.random(),
```
Comment (archiewood) [inline packages/preprocess/src/process-queries.cjs:272]: is this what we want? Any issue with passing a random id?

**Comment by archiewood** [comment]
This appears to be breaking the windows tests

**Comment by csjh** [comment]
Seems like the tests are already failing in `features/a`

File: packages/preprocess/src/process-queries.cjs (hunk 1 above)
Comment (csjh) [reply packages/preprocess/src/process-queries.cjs:272]: Shouldn't be an issue, just meant to have some sort of differentiator between toasts

File: packages/preprocess/src/process-queries.cjs [hunk 2]
```diff
@@ -261,6 +267,15 @@ const createDefaultProps = function (filename, componentDevelopmentMode, duckdbQ
...
+				title: info.id,
+				message: \`Query \${info.id} is loading...\`,
+				status: 'info'
```
Comment (mcrascal) [inline packages/preprocess/src/process-queries.cjs:275]: I think the behaviour we want on the toast isn't several toasts saying "query x is loading..." but rather just a single "loading..." toast that's present until all long-running queries are done.

**Review by mcrascal — CHANGES_REQUESTED** [review]

**Pull request closed without merging**